Update/init.txt

```diff
@@ -68,19 +70,20 @@ void main()
 		SetGlobalFlag(GEffectExtendMaxNum , 0);
 		SetGlobalFlag(GAltBGMflowMaxNum , 0);
 		SetGlobalFlag(GAltSEflowMaxNum , 0);
+		SetGlobalFlag(GArtStyle , 1);
```

Comment:
Hmm maybe I should remove these again? Console arcs don't support CGAlt anyway, right?

Reply:
It's probably safe to leave them there.  MG sprites are not currently supported, but I'd like to add them some day.